assets/js/metric_config.js

```diff
@@ -209,7 +210,9 @@ function update_chart_data(chart_obj,x_axis_data ,points, point_name){
   
 function update_guage_data(guage_obj , points , point_name )
 {
-  var data = points[point_name];
+  var data = (points[point_name]).toFixed(0);
```

Comment:
I have used .toFixed in metric_config.js. (not tapering the original values being received)
-the gauge is dynamic , toFixed(0) because the axis dial will end up the lots of decimals  even starting with integer values. As gauge get divided dynamically to
5 division based on the max value.
Conditions : 
- Less traffic on network will result in 0 being displayed.